src/OpenTelemetry.Instrumentation.Hangfire/README.md

```diff
@@ -0,0 +1,26 @@
+# Hangfire Instrumentation for OpenTelemetry .NET
+
+Automatically instruments the Hangfire jobs from
+[Hangfire](https://www.nuget.org/packages/Hangfire/).
+
+
+## Configuration
```

Comment:
Consider following the structure from https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore

Main issue with current doc is - it does not hint that using Services.AddOpenTelemetryTracing requires additional nuget installations.